Dashboard/Views/ScrutineeringView.axaml.cs

```diff
@@ -19,4 +42,57 @@ public void Previous(object source, RoutedEventArgs args)
     {
         Slides.Previous();
     }
+
+    private void OnPass(object sender, RoutedEventArgs e)
+    {
+        OnStepClick(sender, true);
+    }
+
+    private void OnFail(object sender, RoutedEventArgs e)
+    {
+        OnStepClick(sender, false);
+    }
+
+    /// <summary>
+    ///     When the Pass/Fail button is clicked, set that step to passed/failed and log to the console as well.
+    /// </summary>
+    private void OnStepClick(object sender, bool isPassed)
```

Comment:
Would it be possible to have this check if the button is already passed/failed? Currently it will trigger the onClick event even if it was already passed/failed so you can have the below occur.
![image](https://github.com/user-attachments/assets/c58b21e9-7058-4891-b757-5168e62785b4)

Reply:
thats a good idea, like if its already passed dont do anyhting but still allow the user to switch between pass and fail?

Reply:
Ya, just a simple if statement like 
`If (StepInList is Already Passed/Failed) Return`
I think it could be something like based on the current code
`if (AllStepsList.Items.OfType<TextBlock>().FirstOrDefault(tb => tb.Name == step.Id).Text.Contains(isPassed ? "Passed" : "Failed")) 
{
 Return
  }`

Reply:
oh lovely! thanks

Reply:
d029de809b00fd0abc9cac4ac7c661c8e31c7416 it worked with this :>